src/pages/project/{sanityProject.title}.jsx

```diff
@@ -1,12 +1,213 @@
 import React from 'react'
 import { graphql } from 'gatsby'
 import Layout from '../../components/Layout'
+import Button from '../../components/Button'
+import IconTile from '../../components/IconTile'
+import calendardays from '../../images/icons/calendardays.svg'
+import alarmclock from '../../images/icons/alarmclock.svg'
+import check from '../../images/icons/check.svg'
+import history from '../../images/icons/history.svg'
+import VolunteerTile from '../../components/VolunteerTile'
+import cardplaceholder from '../../images/card-placeholder.svg'
+import Accordion from '../../components/Accordion'
+import { GatsbyImage, getImage } from 'gatsby-plugin-image'
+import styled from 'styled-components'
+import CardBlock from '../../components/CardBlock'
+import Card from '../../components/Card'
+import CardPlaceholder from '../../images/card-placeholder.svg'
 
+
+const PrimaryContent = styled.div.attrs({
+  className: 'width-wrapper',
+})`
+  display: flex;
```

Comment:
let's add `flex-direction: row-reverse` so that right actually shows up on right and left on left.